services/API-service/src/api/data/data.model.ts

```diff
@@ -135,11 +135,26 @@ export class TriggeredArea {
   public population_affected: number;
 }
 
-export class DisasterEvent {
+export class CountryEvent {
   public country_code: string;
-  public start_date: Date;
-  public end_date?: Date;
-  public id: number;
+  public start_date: string;
```

Comment:
JavaScript convention is [camelCase](https://news.ycombinator.com/item?id=6777600)